js/main.js

```diff
@@ -46,7 +46,7 @@ if (shouldQuit) {
 
 var symphonyAutoLauncher = new AutoLaunch({
     name: 'Symphony',
-    path: process.execPath,
+    path: isMac ? '/Applications/' + app.getName() + '.app' : process.execPath,
```

Comment:
we are relying on app being installed is this exact location - which it will but in case it is put elsewhere - couldn't you instead use app.getPath('exe') and figure out from there?